### PR TITLE
added game functionality

### PR DESCRIPTION
**Pull request by jspicuzza21**
Added functionality of adding player to a random game session upon search and also when a player creates a game session.

### CODE REVIEW

File: src/client/store/thunks/gameThunks.js [hunk 1]
```diff
@@ -14,13 +14,26 @@ export const playAsGuestThunk = (name) => (dispatch) => {
...
       console.log(e);
     });
 };
+
+export const findRandomGame = () => () => {
+  return axios
+    .get('/game/gameSession')
+      .then(({data}) => {
+        const {id} = data
+        axios.put(`/user/session`, {id})
```
Comment (kwons90) [inline src/client/store/thunks/gameThunks.js]: I believe the server currently works with params:
`// adds users to a game session
userRouter.put("/session/:id", async (req, res) => {
  try {
    const {id} = req.params;
    const { gameSessionId } = req.body;
    await Session.update({gameSessionId}, {where : {id}})
    res.sendStatus(200)
  } catch (e) {
    res.sendStatus(500);
    console.log(e);
  }
})`

Comment (jspicuzza21) [reply src/client/store/thunks/gameThunks.js]: can you elaborate?

Comment (kwons90) [reply src/client/store/thunks/gameThunks.js]: You're sending id as a req.body to the session route, but put route for /user/session takes id as a parameter through user/session/:id

**Review by kwons90 — DISMISSED** [review]
Made the change and otherwise looks good

**Comment by jspicuzza21** [comment]
Unless you changed the server route it's not going to work.

**Review by kwons90 — APPROVED** [review]

**Pull request merged**